PsfLauncher/StartProcessHelper.h

```diff
@@ -49,6 +48,12 @@ HRESULT StartProcess(LPCWSTR applicationName, LPWSTR commandLine, LPCWSTR curren
         "ERROR: Failed to create a process for %ws",
         applicationName);
 
+    if (exitCode != nullptr)

```

Comment:
The process executes only if exitCode isn't null?  I can pass in any number as long as it isn't null?  How does passing in a number tell the dveloper that "This process will run as long as you pass in any number.?"

Reply:
This purpose of this condition (exitCode != nullptr) is to collect the exitCode only when caller needs the exitCode and passes in a valid pointer. passing in a nullptr to GetExitCodeProcess API would result in error.

"This process will run as long as you pass in any number.?" I didn't get how its related to running our process. CreateProcess is still done irrespective of this condition. Could you please elaborate?